### PR TITLE
chore(install): remove observability packs installation

### PR DESCRIPTION
**Pull request by jbeveland27**
https://newrelic.atlassian.net/browse/VIRTUOSO-1318

### CODE REVIEW

File: internal/install/recipe_installer.go [hunk 1]
```diff
@@ -294,32 +294,10 @@ func (i *RecipeInstaller) install(ctx context.Context) error {
...
-	if err = i.fetchAndInstallPacks(ctx, recipesForInstall); err != nil {
-		return err
-	}
-
 	log.Debugf("Done installing.")
 	return nil
 }
 
-func (i *RecipeInstaller) fetchAndInstallPacks(ctx context.Context, recipesForInstall []types.OpenInstallationRecipe) error {
-	packs, err := i.packsFetcher.FetchPacks(ctx, recipesForInstall)
```
Comment (natecanfield822) [inline internal/install/recipe_installer.go:306]: I don't see this used anyplace else, should the packsFetcher code also be removed?

File: internal/install/recipe_installer.go [hunk 2]
```diff
@@ -294,32 +294,10 @@ func (i *RecipeInstaller) install(ctx context.Context) error {
...
-	if err != nil {
-		// nolint: golint
-		return fmt.Errorf("Failed to fetch observability packs: %s", err)
-	}
-	log.Debugf("Fetched Packs: %d", len(packs))
-
-	if len(packs) > 0 {
-		if err := i.packsInstaller.Install(ctx, packs); err != nil {
```
Comment (natecanfield822) [inline internal/install/recipe_installer.go:314]: I don't see this used anyplace else, should the packsInstaller code also be removed?

**Review by natecanfield822 — COMMENTED** [review]
Left some feedback

**Review by Julien4218 — APPROVED** [review]

**Comment by Julien4218** [comment]
@jbeveland27 is there a separate story to remove all the type definitions related to pack?
For example in types/recipe.go, look for pack.

**Comment by jbeveland27** [comment]
@Julien4218 There is not. I believe that should be a fairly automatic update to `.tutone.yml` and then run codegen again. I can add a follow up ticket for that.

I'm going to push another update in a moment to remove the `packs` directory altogether. It's dead code at this point.

**Review by Julien4218 — APPROVED** [review]

**Pull request merged**